docs/src/pages/css-in-js/api/api-de.md

```diff
@@ -243,14 +237,14 @@ import ReactDOM from 'react-dom';
 import { StylesProvider } from '@material-ui/styles';
 
 function App() {
-  return (
-    &lt;StylesProvider jss={jss}&gt;...&lt;/StylesProvider&gt;
-  );
+return (
+&lt;StylesProvider jss={jss}&gt;...&lt;/StylesProvider&gt;
+);
 }
```

Comment:
Oops.

Reply:
Well that's nice though. Prettier helped us detect broken markdown in the translated files: https://next.material-ui.com/css-in-js/api/#themeprovider

The german translation has a broken props table and doesn't highlight the code properly.

Reply:
It looks like all of the translations have broken tables and headers:

https://next.material-ui.com/zh/css-in-js/api/#stylesprovider
https://next.material-ui.com/ru/css-in-js/api/#stylesprovider
https://next.material-ui.com/pt/css-in-js/api/#stylesprovider
https://next.material-ui.com/fr/css-in-js/api/#stylesprovider
https://next.material-ui.com/es/css-in-js/api/#stylesprovider
https://next.material-ui.com/de/css-in-js/api/#stylesprovider

Reply:
We should probably report the problem to Crowdin. It seems they fail to handle the table format correctly. This is interesting: `/docs/src/pages/system/api/api.md` is handled correctly. Maybe it's linked to the usage of HTML elements inside the markdown table 🤔.